<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="47" started-at="2019-01-08T10:43:02Z" finished-at="2019-01-08T10:43:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="47" started-at="2019-01-08T10:43:02Z" finished-at="2019-01-08T10:43:02Z">
      <class name="BeforeTest.BeforeMethod_Test">
        <test-method status="PASS" signature="Test1()[pri:1, instance:BeforeTest.BeforeMethod_Test@653222]" name="Test1" duration-ms="16" started-at="2019-01-08T10:43:02Z" finished-at="2019-01-08T10:43:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
        <test-method status="SKIP" signature="Test2()[pri:2, instance:BeforeTest.BeforeMethod_Test@653222]" name="Test2" duration-ms="0" started-at="2019-01-08T10:43:02Z" finished-at="2019-01-08T10:43:02Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping Test 2 ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping Test 2 
at BeforeTest.BeforeMethod_Test.Test2(BeforeMethod_Test.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test2 -->
      </class> <!-- BeforeTest.BeforeMethod_Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
